test/gurka/test_ferry_connections.cc

```diff
@@ -297,7 +297,8 @@ TEST(Standalone, ReclassifyFerryUntagDestOnly) {
   EXPECT_TRUE(std::get<1>(upclassed)->classification() == valhalla::baldr::RoadClass::kPrimary);
 
   // we expect to take the shorter route on the left and the detour on the right
-  for (const std::string& mode : {"auto", "motorcycle", "taxi", "bus", "hov", "truck"}) {
+  std::vector<std::string> modes = {"auto", "motorcycle", "taxi", "bus", "hov", "truck"};
+  for (const auto& mode : modes) {
```

Comment:
@cvvergara curious: is it the implicit conversion of `const char` to `std::string` that's raising some warning or this is just for style?

Reply:
This is an array of `char*` 
```
{"auto", "motorcycle", "taxi", "bus", "hov", "truck"}
```
I opted for creating a variable, instead of casting the array on the loop for clarity